frontend/src/components/Header.tsx

```diff
@@ -0,0 +1,224 @@
+    [theme.breakpoints.up('lg')]: {
+      display: 'none'
+    }
+  },
+  logo: {
+    display: 'none',
+    height: 60,
+    padding: theme.spacing(),
+    paddingLeft: 0,
+    [theme.breakpoints.up('sm')]: {
+      display: 'block'
+    }
+  },
+  spacing: {
+    flexGrow: 1
+  },
+  activeLink: {
+    '&:after': {
+      content: "''",
+      position: 'absolute',
+      bottom: 0,
+      left: 0,
+      width: '100%',
+      height: 2,
+      backgroundColor: 'white'
+    }
+  },
+  activeMobileLink: {
+    fontWeight: 700,
+    '&:after': {
+      content: "''",
+      position: 'absolute',
+      top: 0,
+      bottom: 0,
+      left: 0,
+      height: '100%',
+      width: 2,
+      backgroundColor: theme.palette.primary.main
+    }
+  },
+  link: {
+    position: 'relative',
+    color: 'white',
+    textDecoration: 'none',
+    margin: `0 ${theme.spacing()}px`,
+    padding: theme.spacing(),
+    borderBottom: '2px solid transparent'
+  },
+  userLink: {
+    display: 'flex',
+    alignItems: 'center',
+
+    '& svg': {
+      marginRight: theme.spacing()
+    }
+  },
+  mdNav: {
+    display: 'none',
+    [theme.breakpoints.up('md')]: {
+      display: 'block'
+    }
+  },
+  lgNav: {
+    display: 'none',
+    [theme.breakpoints.up('lg')]: {
+      display: 'block'
+    }
+  },
+  mobileNav: {
+    padding: `${theme.spacing(2)}px ${theme.spacing()}px`
+  }
+}));
+
+interface NavItem {
+  title: string;
+  path: string;
+  users: number;
+}
+
+export const Header: React.FC = () => {
+  const classes = useStyles();
+  const { currentOrganization, user } = useAuthContext();
+  const [navOpen, setNavOpen] = useState(false);
+
+  let userLevel = 0;
+  if (user && user.isRegistered) {
+    if (user.userType === 'standard') {
+      if (currentOrganization?.userIsAdmin) {
+        userLevel = ORG_ADMIN;
+      } else {
+        userLevel = ORG_USER;
+      }
+    } else {
+      userLevel = GLOBAL_ADMIN;
+    }
+  }
+
+  const navItems: NavItem[] = [
+    { title: 'Vulnerabilities', path: '/vulnerabilities', users: ALL_USERS },
+    { title: 'Risk Summary', path: '/risk', users: ALL_USERS },
+    {
+      title: 'Organization Settings',
+      path: '/organization',
+      users: ORG_ADMIN | GLOBAL_ADMIN
+    },
+    {
+      title: 'My Organizations',
+      path: '/organizations',
+      users: ORG_USER | ORG_ADMIN
+    },
+    {
+      title: 'Manage Organizations',
+      path: '/organizations',
+      users: GLOBAL_ADMIN
+    },
+
+    { title: 'Scans', path: '/scans', users: ORG_ADMIN | GLOBAL_ADMIN },
+    { title: 'Manage Users', path: '/users', users: GLOBAL_ADMIN }
+  ].filter(({ users }) => (users & userLevel) === userLevel);
+
+  const desktopNavItems: JSX.Element[] = navItems.map(({ title, path }) => (
+    <NavLink
+      to={path}
+      key={title}
+      activeClassName={classes.activeLink}
+      className={classes.link}
+    >
+      {title}
+    </NavLink>
+  ));
+
+  return (
+    <div>
+      <AppBar position="static">
```

Comment:
Also, why purple for the header? Could we keep it dark blue to match the footer, perhaps?

![image](https://user-images.githubusercontent.com/1689183/92181048-60444700-ee16-11ea-94dc-85f78eed9294.png)

Or maybe light blue to match the figma designs

![image](https://user-images.githubusercontent.com/1689183/92181117-810c9c80-ee16-11ea-9550-068f3f5b6527.png)

Reply:
Made primary theme color #28A0CB from figma and updated footer as well